asif.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:  2.2
 name:           asif
-version:        6.0.1
+version:        6.0.2
```

Comment:
This change alone probably doesn't need a version bump

Reply:
Nevermind. I can't be bothered fixing whatever build error this was throwing up. It was just something I incidentally noticed while working on other stuff. It can get fixed next time someone is properly working in this library.